jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java

```diff
@@ -127,6 +127,7 @@
    private String kafkaTopicOdeBsmTxPojo = "topic.OdeBsmTxPojo";
    private String kafkaTopicOdeBsmDuringEventPojo = "topic.OdeBsmDuringEventPojo";
    private String kafkaTopicFilteredOdeBsmJson = "topic.FilteredOdeBsmJson";
+	private String kafkaTopicOdeRawEncodedMessageJson = "topic.OdeRawEncodedMessageJson";
```

Comment:
Fix spacing on line begin to match file standard

Reply:
Addressed